.github/workflows/rust.yml

```diff
@@ -0,0 +1,22 @@
+name: Rust
+
+on:
+  push:
+    branches: [ master ]
+  pull_request:
+    branches: [ master ]
+
+env:
+  CARGO_TERM_COLOR: always
+
+jobs:
+  build:
+
+    runs-on: ubuntu-latest
+
+    steps:
+    - uses: actions/checkout@v2
+    - name: Build
+      run: cargo build --verbose
+    - name: Run tests
+      run: cargo test --verbose
```

Comment:
Consider adding a `cargo fmt -- --check` if we want to use `rustfmt`. If we do add it, then we need it as a dependency.

Reply:
I think this is the right dependency?

https://github.com/marketplace/actions/rust-toolchain

Reply:
Okay, gottem.